src/benchmarks/gc/src/commonlib/bench_file.py

```diff
@@ -1137,6 +1153,14 @@ def __str__(self) -> str:
         return str(self.test_status_or_trace_path)
 
 
+# @with_slots
```

Comment:
Is this unused? Probably should delete or comment why this is here.

Reply:
this is used in this file, which I presume was the reason why it got move here from analysis.

---
In reply to: [355727310](https://github.com/dotnet/performance/pull/1086#discussion_r355727310) [](ancestors = 355727310)